plugins/hotkeyshortcuts.koplugin/defaults.lua

```diff
@@ -0,0 +1,117 @@
+-- typed on my Commodore 64 ;)
```

Comment:
Too Much Information :)

Reply:
too bad. ;)

Reply:
Still there.

Reply:
Correct, and we are still talking about it.